src/reliability.jl

```diff
@@ -0,0 +1,78 @@
+using LinearAlgebra: isposdef
+
+struct Reliability{T <: Real}
+    alpha::T
+    dropped::Vector{Pair{Int, T}}
+end
+
+function Base.show(io::IO, x::Reliability)
+    println(io, "Reliability for all items: $(round(x.alpha; digits=4))")
```

Comment:
I suggest using `@printf` to ensure that the printed value always has 4 digits. Using `round`, fewer digits will be printed they are enough (like for 1.0), which is less clear for users:
```suggestion
    @printf(io, "Reliability for all items: %.4f", x.alpha)
```
(Also needs `using Printf` somewhere if that's not already loaded.)

Reply:
It is already in `StatsBase.jl`